e2e/cypress/integration/organizations.spec.js

```diff
@@ -21,16 +22,13 @@ describe('Organization', () => {
         cy.contains('0')
         cy.get('a[href="/organization/cypress-test-org"]')
     })
+    it.skip('can use api to create an organization with extras', () => {
+        cy.create_dcat_org(dcatOrg);
+    });
```

Comment:
I think this is acting as intended, no need to change. It's interesting functionality; it comes out weird in the command line log report but it's easy to clarify that. You may want to add a comment as to why it's being skipped though!